fsh/input/fsh/DentalCare-Bundle-Scenario-2.fsh

```diff
@@ -210,4 +222,53 @@ Usage: #example
   * city = "Leiderdorp"
   * postalCode = "2353 GA"
   * country = "Nederland"
-    * extension[http://nictiz.nl/fhir/StructureDefinition/ext-CodeSpecification].valueCodeableConcept = urn:iso:std:iso:3166#NL "Netherlands"
\ No newline at end of file
+    * extension[http://nictiz.nl/fhir/StructureDefinition/ext-CodeSpecification].valueCodeableConcept = urn:iso:std:iso:3166#NL "Netherlands"
+
+Instance: DentalCare-Payer-Jansen
+InstanceOf: http://nictiz.nl/fhir/StructureDefinition/nl-core-Payer.InsuranceCompany
+Usage: #example
+* text
+  * status = #generated
+  * div = "<div xmlns='http://www.w3.org/1999/xhtml'>Verzekering: Menzis<br/>Patiënt: Anita Jansen<br/>begin datum: 2025-01-01<br/>Status: Actief<br/>Betaler: Menzis Zorgverzekeraar N.V.</div>"
+* status = #active
+* type = $CoverageTypeCodeSystemOID#AT "Aanvullend + tand"
+* subscriberId = "01234567"
+* beneficiary = Reference(DentalCare-Patient-Jansen) "Anita Jansen"
+* period
+  * start = "2025-01-01"
+  * end = "2026-01-01"
+* payor = Reference(DentalCare-Organization-Menzis) "Menzis Zorgverzekeraar N.V."
+
+Instance: DentalCare-Organization-Menzis
+InstanceOf: http://nictiz.nl/fhir/StructureDefinition/nl-core-Payer-Organization
+Usage: #example
+* text
+  * status = #generated
+  * div = "<div xmlns='http://www.w3.org/1999/xhtml'>Organisatie: Menzis Zorgverzekeraar<br/>Adres: Achillesstraat 12, 1234 AA Amsterdam</div>"
+* name = "Menzis Zorgverzekeraar N.V."
+* address
```

Comment:
Voor Menzis moet een ander adres gekozen worden dan voor de Patient (ook aanpassen in narrative van deze resource).